### ISSUE TITLE
Consider disabling initial jupyterlab news prompt

### ISSUE THREAD

**Issue by ryanlovett** [berkeley-dsep-infra/hubs#4250]
### Bug description

The first time users start lab on stat159 hub, a message box appears at the bottom right of the screen asking, "Would you like to receive official Jupyter news?"

<img width="366" alt="Screenshot 2023-02-10 at 1 20 15 PM" src="https://user-images.githubusercontent.com/10248125/218199608-7e0eed95-d818-4c87-9f93-1d7ed272a490.png">

When the user disables the message, a file gets written to `~/.jupyter/lab/user-settings/\@jupyterlab/apputils-extension/notification.jupyterlab-settings` with the content:
```
{
    // Notifications
    // @jupyterlab/apputils-extension:notification
    // Notifications settings.
    // *******************************************

    // Fetch official Jupyter news
    // Whether to fetch news from Jupyter news feed. If `true`, it will make a request to a website.
    "fetchNews": "false"
}
```

The user is not prompted on future lab launches.

### Environment & setup

- Hub: stat159

### How to reproduce

Login to stat159 hub and start lab for the first time.

**Comment by ryanlovett** [comment]
I spoke with @fperez in person about it and he agreed that it should be muted in this instructional setting.

[Binder docs](https://jupyterlab.readthedocs.io/en/stable/user/binder.html#override-default-settings) suggest configuring this by creating an `overrides.json` file with:
```
{
  "@jupyterlab/apputils-extension:notification": {
    "fetchNews": "false"
  }
}
```
and putting it in `${NB_PYTHON_PREFIX}/share/jupyter/lab/settings/`.

**Comment by fperez** [comment]
Agreed, thanks @ryanlovett !

**Comment by ryanlovett** [comment]
Fixed by #4253.
